taggit/models.py

```diff
@@ -157,7 +157,7 @@ def tags_for(cls, model, instance=None, **extra_filters):
 
 
 class GenericTaggedItemBase(CommonGenericTaggedItemBase):
-    object_id = models.IntegerField(verbose_name=_("object ID"), db_index=True)
+    object_id = models.BigIntegerField(verbose_name=_("object ID"), db_index=True)
```

Comment:
this will break backward compatibility

Reply:
I see no other way to fix this issue  : (

Reply:
They could do a version check of Django before migrating.
And use:

```shell
In [1]: import django
In [2]: django.VERSION
Out [2]: (3, 1, 1, 'final', 0)
```

Reply:
probably moving to abstract base model is the best futuristic approach